go.mod

```diff
@@ -2,17 +2,19 @@ module github.com/thirdknife/scoutingapp
 
 go 1.22.3
 
-require github.com/labstack/echo/v4 v4.12.0
+require (
```

Comment:
ran `go mod tidy`

Reply:
This seems fine, but I'm not sure why it moved dependencies from indirect to direct. The golang changes in this branch shouldn't have required that. Anyway, probably fine.

Reply:
Yeah, it shouldn't have moved it to direct.